.github/workflows/integration_test_8gpu_amd.yaml

```
@@ -0,0 +1,39 @@
+name: 8 AMD GPU Integration Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  schedule:
+    # Runs every 6 hours
+    - cron: '0 */6 * * *'
+concurrency:
+  group: unit-test${{ github.workflow }}-${{ github.ref == 'refs/heads/main' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l -eo pipefail {0}
+
+jobs:
+  build-test:
+    uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@main
+    with:
+      runner: linux.rocm.gpu.mi300.8
+      gpu-arch-type: cuda
```

```suggestion
      gpu-arch-type: rocm
```

Needed to skip certain steps in [linux_job_v2.yml](https://github.com/pytorch/test-infra/blob/abe305c3a976c7108ac60c9715e063e42c224807/.github/workflows/linux_job_v2.yml#L164) that do not work on ROCm CI runners.